pom.xml

```diff
@@ -298,7 +298,7 @@
             <dependency>
                 <groupId>org.hibernate.validator</groupId>
                 <artifactId>hibernate-validator</artifactId>
-                <version>7.0.5.Final</version>
+                <version>8.0.1.Final</version>
```

Comment:
This version of `hibernate-validator` does not support `java 8`. Applying this upgrade will mark the SDK's support to `Java 8` obsolete.

https://github.com/ExpediaGroup/expediagroup-java-sdk/pull/257